code/modules/religion/rites.dm

```diff
@@ -0,0 +1,223 @@
+	for(var/i in ritual_invocations)
+		if(first_invoke) //instant invoke
+			user.say(i)
+			first_invoke = FALSE
+			continue
+		if(!ritual_invocations.len) //we divide so we gotta protect
+			return FALSE
+		if(!do_after(user, target = user, delay = ritual_length/ritual_invocations.len))
+			return FALSE
+		user.say(i)
+	if(!do_after(user, target = user, delay = ritual_length/ritual_invocations.len)) //because we start at 0 and not the first fraction in invocations, we still have another fraction of ritual_length to complete
+		return FALSE
+	if(invoke_msg)
+		user.say(invoke_msg)
+	return TRUE
+
+
+///Does the thing if the rite was successfully performed. return value denotes that the effect successfully (IE a harm rite does harm)
+/datum/religion_rites/proc/invoke_effect(mob/living/user, atom/religious_tool)
+	GLOB.religious_sect.on_riteuse(user,religious_tool)
+	return TRUE
+
+
+/*********Technophiles**********/
+
+/datum/religion_rites/synthconversion
+	name = "Synthetic Conversion"
+	desc = "Convert a human-esque individual into a (superior) Android."
+	ritual_length = 1 MINUTES
+	ritual_invocations = list("By the inner workings of our god...",
+						"... We call upon you, in the face of adversity...",
+						"... to complete us, removing that which is undesirable...")
+	invoke_msg = "... Arise, our champion! Become that which your soul craves, live in the world as your true form!!"
+	favor_cost = 500
+
+/datum/religion_rites/synthconversion/perform_rite(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+		return FALSE
+	var/atom/movable/movable_reltool = religious_tool
+	if(!movable_reltool)
+		return FALSE
+	if(!LAZYLEN(movable_reltool.buckled_mobs))
+		. = FALSE
+		if(!movable_reltool.can_buckle) //yes, if you have somehow managed to have someone buckled to something that now cannot buckle, we will still let you perform the rite!
+			to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+			return
+		to_chat(user, "<span class='warning'>This rite requires an individual to be buckled to [movable_reltool].</span>")
+		return
+	return ..()
+
+/datum/religion_rites/synthconversion/invoke_effect(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		CRASH("[name]'s perform_rite had a movable atom that has somehow turned into a non-movable!")
+	var/atom/movable/movable_reltool = religious_tool
+	if(!movable_reltool?.buckled_mobs?.len)
+		return FALSE
+	var/mob/living/carbon/human/human2borg
+	for(var/i in movable_reltool.buckled_mobs)
+		if(istype(i,/mob/living/carbon/human))
+			human2borg = i
+			break
+	if(!human2borg)
+		return FALSE
+	human2borg.set_species(/datum/species/android)
+	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
+	return TRUE
```

Comment:
```suggestion
	return ..()
```

I want to see this convention happen before someone gets a headache on why `on_riteuse` doesn't work.